src/__experimental__/components/simple-color-picker/color-option/color-option.component.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import baseTheme from '../../../../style/themes/base';
+import tagComponent from '../../../../utils/helpers/tags';
+import StyledColorOption from './style/color-option.style';
+import ColorSampleBox from '../color-sample-box';
+import StyledColorOptionInput from '../color-option-input/style/color-option-input.style';
+
+const ColorOption = (props) => {
+  return (
+    <StyledColorOption
+      color={ props.color }
+      checked={ props.checked }
+      className={ props.className }
+      aria-checked={ props.checked }
+      role='radio'
+      { ...props }
+      { ...tagComponent('color-option', props) }
+    >
+      <StyledColorOptionInput
+        className={ props.className }
+        onChange={ props.onChange }
+        checked={ props.checked }
+        name={ props.name }
+        type='radio'
+        value={ props.color }
+        id={ props.optionId }
+        aria-label={ props.name }
+      />
+      <ColorSampleBox color={ props.color } checked={ props.checked } />
+    </StyledColorOption>
+  );
+};
+
+ColorOption.defaultProps = {
+  checked: false,
+  onChange: null,
+  theme: baseTheme
+};
+
+ColorOption.propTypes = {
+  /** the value of the color that is represented by this ColorOption */
+  color: PropTypes.string,
```

Comment:
shouldn't there be a colors list from the `OptionsHelper`?

Reply:
not really, in this component users can use any colors they want to